Mailer/app/api/Validation/schemas/mail-types.schema.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Mail types resource",
+  "type": "object",
+  "properties": {
```

Comment:
In schemas, don't allow extra properties. Let the API be strict on what can be requested.

Reply:
So all used properties should be required?

Reply:
yes (didn't know the name of the property without searching for it)